src/labels/CustomLabels.labels

```diff
@@ -9875,6 +9875,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Error shown when errors must be removed before something else can occur.</shortDescription>
         <value>Clear errors before proceeding.</value>
     </labels>
+    <labels>
+        <fullName>stgConcurrentBatchError</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgConcurrentBatchError</shortDescription>
```

Comment:
Also, as @melissabarber  indicated, the short description should be a descriptive text for the label. 
@sam-knox ^

Reply:
@RadSFDeveloper I was looking for @melissabarber's comment earlier, but couldn't find it in Github, so I added one of my own above.